### DISCUSSION TITLE
Groups, collections or favorites for Podcasts

### DISCUSSION THREAD

**Discussion by Dragonatorul** [advplyr/audiobookshelf#1898] [Ideas]
It would be useful to have a feature similar to Collections (for Audiobooks), but for Podcasts. I am following multiple types of podcasts: Infosec, programming, lifestyle, etc., and would like to be able to group them by interest, with a group specifically for the ones I actively follow. 

I would also love to have a "Favorites" section on the Podcasts home page. Honestly, at this point other than the "Continue Listening" section, the rest of the home page is pretty much useless to me because of the spam from all the podcasts I follow but only occasionally listen to.

**Comment by advplyr** [comment]
Yeah the home page was designed for audiobooks and when podcasts got added they were just sort of wedged in there. There are plans to improve the home page across the board.

What you are describing for your types of podcasts sounds like it fits better as a tag. Are you suggesting collections because the tag filter is not as convenient?

**Reply by Dragonatorul** [reply]
I agree, the tags seem like the best fit for this exact requirement, but you're right, they are a bit convoluted to use. The groups in audiobooks provide a friendly at-a-glance interface to select a particular group/tag. Whereas the tags need to be specifically filtered with multiple clicks. 

I suppose bookmarks could be set for each specific filter, since the filter appears to be reflected in the URL (correct me if I'm wrong), but that is still external to the app. 

Perhaps a way to pin tags to the homepage would work better, or a section similar to groups, but based around tags. Either way, I strongly believe that something to make easily grouping podcasts, and accessing the groups easily is needed.

**Reply by advplyr** [reply]
Setting custom home page shelves is the plan so the idea would be you can create any filter/sort as a home page shelf. That may be a little while though since I'm currently working on a big database migration

**Reply by Dragonatorul** [reply]
That sounds great. I'm in no rush, just wanted to start a discussion to put this on your radar.

Unrelated, but are there plans to support dedicated databses like mariadb or postgres?

**Reply by advplyr** [reply]
I'm working on moving the database from a JSON file db to a sqlite db. Once that is done we can start thinking about supporting external db

**Reply by undaunt** [reply]
I'm wondering if it would be possible and maybe simpler to create the concept of a favorited episode with a boolean flag in the database structure. As I'm migrating away from Pocket Casts that's one feature I really enjoy. Eg: In a 300+ episode TTRPG storyline, going back to some of my favorite moments quickly by filtering on favorites, or perhaps in the future even using it as a criteria for playlist building if we were able to create multi-podcast playlists?

**Comment by manuelmgn** [comment]
Hello, Sorry if I'm reviving an old post, but I wanted to make a suggestion that I think fits here.

It's about the possibility for **each user to mark podcasts as favorites**, follow them, or subscribe (whatever terminology is used). I believe this would be a very interesting feature.

I use Audiobookshelf with several people mainly for podcasts, and while we have similar tastes, not everyone enjoys the same podcasts. It can be a hassle to see new episodes in "Latest" that don't interest everyone.

Visually, I think this could be resolved by adding, for example, a heart or a star in the bottom left corner of each podcast in the Library, and in Latest, there could be a quick filter to show only favorites or all.

Thanks! And also, I'd like to appreciate the great work of the Audiobookshelf team.
